core/templates/services/speech-synthesis-chunker.service.ts

```diff
@@ -173,31 +184,35 @@ export class SpeechSynthesisChunkerService {
     var elt = $('<div>' + html + '</div>');
     // Convert links into speakable text by extracting the readable value.
     elt.find('oppia-noninteractive-' + this.RTE_COMPONENT_NAMES.Link)
-      .replaceWith(function() {
+      .replaceWith(function(this: HTMLElement) {
```

Comment:
Can you confirm if `this: HTMLElement` is required ? Here and below.

Reply:
yes, it is since otherwise the type of `this` will not be defined.

Reply:
Not required, I have tested this.